pkg/auth/authenticator/password/keystonepassword/keystonepassword_test.go

```diff
@@ -24,50 +24,20 @@ func TestKeystoneLogin(t *testing.T) {
 
 	const ID = "0123456789"
 
-	th.Mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, `
-			{
-				"versions": {
-					"values": [
-						{
-							"status": "stable",
-							"id": "v3.0",
-							"links": [
-								{ "href": "%s", "rel": "self" }
-							]
-						},
-						{
-							"status": "stable",
-							"id": "v2.0",
-							"links": [
-								{ "href": "%s", "rel": "self" }
-							]
-						}
-					]
-				}
-			}
-		`, th.Endpoint()+"v3/", th.Endpoint()+"v2.0/")
-	})
-
 	th.Mux.HandleFunc("/v3/auth/tokens", func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Add("X-Subject-Token", ID)
 		type AuthRequest struct {
-			Query string `json:"query"`
```

Comment:
unmarshal ignores fields that don't exist in the struct, so if you don't need to read it, you can leave it out